app/move/steps/create.js

```diff
@@ -16,32 +15,48 @@ module.exports = {
     resetJourney: true,
     skip: true,
     checkJourney: false,
-    next: 'pnc-search',
+    next: [
+      {
+        fn: req => {
+          return req.session.currentLocation.location_type === 'prison'
+        },
+        next: 'nomis-search',
```

Comment:
I repeat this check on the current location multiple times. It should probably be put in a field in the journey session storage